src/client/components/GlobalState.tsx

```diff
@@ -7,7 +7,7 @@ type GlobalProps = {
   globalState: GlobalState;
 };
 
-export const GlobalStateProvider: FunctionComponent<GlobalProps> = props => {
+export const GlobalStateProvider: FunctionComponent<GlobalProps> = (props) => {
```

Comment:
I've tried looking for `prettier` and `editorconfig` rules to specify bracket style, but I've not had any luck. Is there a way we can force the requirements for brackets for function arguments ?

Reply:
I think they are enforced by prettier. `yarn lint` fails when the brackets aren't present.

Reply:
ah I see, at least it will be consistent going forward 👍

Reply:
we could add `yarn lint` to CI

Reply:
ahh I see, I assumed it was run on a `pre-commit` hook or something. Yeah it should be in CI and a pre-commit hook if possible. Maybe one for another PR tho